crates/consensus-any/src/receipt/envelope.rs

```diff
@@ -16,23 +17,23 @@ use alloy_rlp::{Decodable, Encodable};
 #[derive(Clone, Debug, PartialEq, Eq)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[doc(alias = "AnyTransactionReceiptEnvelope", alias = "AnyTxReceiptEnvelope")]
-pub struct AnyReceiptEnvelope<T = Receipt<Log>> {
+pub struct AnyReceiptEnvelope<T = Log> {
```

Comment:
this could potentially be problematic for use cases where the any network should support different receipt types, for example op.

but this doesn't change the defaults and is fine imo.

we should also add

> AsRef<Self> for alloy_primitives::Log

to alloy_primitives